utils/runtime-fuzzer/src/generator.rs

```diff
@@ -231,3 +231,13 @@ fn arbitrary_limited_bytes(u: &mut Unstructured, limit: usize) -> Result<Vec<u8>
     let arb_size = u.int_in_range(0..=limit)?;
     u.bytes(arb_size).map(|bytes| bytes.to_vec())
 }
+
+fn arbitrary_value(u: &mut Unstructured, current_balance: u128) -> Result<u128> {
+    let (lower, upper) = match u.int_in_range(0..=99)? {
+        0..=4 => (0, EXISTENTIAL_DEPOSIT),
```

Comment:
I'd reduce it
```suggestion
        0..=2 => (0, EXISTENTIAL_DEPOSIT),
```

Reply:
I would also reorder it like this:
- `(0, 0)`
- `(0, EXISTENTIAL_DEPOSIT)`
- `(EXISTENTIAL_DEPOSIT, current_balance),`

Reply:
done